README.md

```diff
@@ -101,11 +114,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
+[x] `$(".post").html()`
 [] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[x] `document.getElementsByClassName("post").innerHTML`
```

Comment:
So close.  2 out of 3.  The last option attempts to call "innerHTML" on an array of elements.  Array does not support "innerHTML".  Option #4 selects all the elements with the class "post", then retrieves the first element in that array, and returns the innerHTML of that element.  Option 1, 3, & 4 are correct.